language/Spanish/strings.po

```diff
@@ -1538,22 +1552,22 @@ msgstr "Posición vertical %1$s"
 #: src/frontend/mame/ui/ui.cpp:1467
 #, c-format
 msgid "Laserdisc '%1$s' Horiz Stretch"
-msgstr "Estiramiento horizontal de Laserdisc «%1$s»"
+msgstr "Estiramiento horizontal de laserdisc «%1$s»"
```

Comment:
Are trademarks not capitalised in Spanish?  Laserdisc is a trademark, and hence treated as a proper noun and capitalised in Germanic languages.  (Honest question, I don't know how it's handled in Spanish.)